app/src/main/java/com/orange/ods/app/ui/components/tabs/ComponentTabs.kt

```diff
@@ -81,14 +87,18 @@ fun ComponentTabs(variant: Variant) {
                 )
                 Subtitle(textRes = R.string.component_tabs_icon_position, horizontalPadding = true)
                 OdsChoiceChipsFlowRow(
-                    selectedChoiceChipIndex = OdsTabRow.Tab.Icon.Position.entries.indexOf(tabsIconPosition.value),
+                    selectedChoiceChipIndex = OdsTabRow.Tab.Icon.Position.entries.indexOf(tabIconPosition.value),
                     modifier = Modifier.padding(horizontal = dimensionResource(id = com.orange.ods.R.dimen.spacing_m)),
                     choiceChips = OdsTabRow.Tab.Icon.Position.entries.map { tabsIconPosition ->
                         val textResId = when (tabsIconPosition) {
                             OdsTabRow.Tab.Icon.Position.Top -> R.string.component_tabs_icon_position_top
                             OdsTabRow.Tab.Icon.Position.Leading -> R.string.component_tabs_icon_position_leading
                         }
-                        OdsChoiceChipsFlowRow.ChoiceChip(stringResource(id = textResId), { this.tabsIconPosition.value = tabsIconPosition }, isTabsIconPositionEnabled)
+                        OdsChoiceChipsFlowRow.ChoiceChip(
+                            stringResource(id = textResId),
+                            { this.tabIconPosition.value = tabsIconPosition },
```

Comment:
```suggestion
                            { this.tabIconPosition.value = tabIconPosition },
```

`tabsIconPosition` can be renamed here too.